index.html

```diff
@@ -8,12 +8,83 @@
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body>
+    <header>
+        <img src="img/karma-logo.svg" alt="Karma logo" >
+        <nav class="topnav" id="myTopnav">
+            <a href="#">Meet Karma</a> 
+            <a href="#">How it works</a>
+            <a href="#">Store</a>
+            <a href="#">Blog</a>
+            <a href="#">Help</a>
+            <a href="#">Login</a>
+          
+            <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+            <i class="fa fa-bars"></i>
+          </a>
+    </nav>
+        
+        
+    </header>
+    <main>
+        <section class="intro">
+            <h1>Introducing Karma</h1>
+            <h2>Bring WIFI with you every where you go</h2>
+            <button>Learn more</button>
+        </section>
+        <section class="items">
+            <h2>Everyone needs a little Karma</h2>
+            <div class="container">
+                <div>
+                    <img src="img/icon-devices.svg" alt="">
+                    <caption><h3>Internet for all devices</h3> </caption>
```

Comment:
```suggestion
                    <p>Internet for all devices</p>
```
`caption` is only used for `table` elements, so it doesn't make sense here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption

Also, this doesn't seem like a heading but instead some text so we can change `h3` to `p`